src/common/Utilities/Util.cpp

```diff
@@ -31,19 +30,14 @@
   #include <arpa/inet.h>
 #endif
 
-static boost::thread_specific_ptr<SFMTRand> sfmtRand;
+static thread_local SFMTRand* sfmtRand = nullptr;
```

Comment:
`unique_ptr`? The block is deallocated on application exit anyway but it could prevent false positives from valgrind.